lib/chai/config.js

```diff
@@ -64,11 +64,31 @@ module.exports = {
    *     chai.config.useProxy = false;  // disable use of Proxy
    *
    * This feature is automatically disabled regardless of this config value
-   * in environments that don`t support proxies.
+   * in environments that don't support proxies.
    *
    * @param {Boolean}
    * @api public
    */
 
-  useProxy: true
+  useProxy: true,
+
+  /**
+   * ### config.proxyExcludedKeys
+   *
+   * User configurable property, defines which non-existing properties should be ignored
+   * instead of throwing an error if they do not exist on the assertion.
+   * This is only applied if the environment Chai is running into supports proxies and
+   * if the `useProxy` configuration setting is enabled.
```

Comment:
Couple nits:

- Line 78: Can remove  "non-existing" since it's redundant with "if they do not exist on the assertion" later on in the same sentence.
- Line 80: "running in" instead of "running into".